PopPUNK/plot.py

```diff
@@ -728,12 +721,64 @@ def outputsForMicroreact(combined_list, coreMat, accMat, clustering, perplexity,
     writeClusterCsv(outPrefix + "/" + os.path.basename(outPrefix) + "_microreact_clusters.csv",
                         combined_list, combined_list, clustering, 'microreact', epiCsv, queryList)
 
+    # write NJ tree
+    if nj_tree is not None:
+        write_tree(nj_tree, outPrefix, "_core_NJ.nwk", overwrite)
+
+    # write MST
+    if mst_tree is not None:
+        write_tree(mst_tree, outPrefix, "_core_MST.nwk", overwrite)
 
     # write the phylogeny .nwk; t-SNE network .dot; clusters + data .csv
-    generate_phylogeny(coreMat, seqLabels, outPrefix, "_core_NJ.nwk", rapidnj, overwrite, threads)
     generate_tsne(seqLabels, accMat, perplexity, outPrefix, overwrite)
 
-def generate_phylogeny(coreMat, seqLabels, outPrefix, tree_suffix, rapidnj, overwrite, threads):
+def write_tree(tree, prefix, suffix, overwrite):
+    """Generate phylogeny using dendropy or RapidNJ
```

Comment:
I think this is incorrect? (is it from copying another docstring?)

Reply:
Corrected in 4e63aca.